modules/gitops-release-notes-1-14-1.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-14.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-14-1_{context}"]
+= Release notes for {gitops-title} 1.14.1
+
+{gitops-title} 1.14.1 is now available on {OCP} 4.13, 4.14, 4.15, 4.16, and 4.17.
+
+[id="errata-updates-1-14.1_{context}"]
+== Errata updates
+
+[id="RHEA-XXXX:NNNN-gitops-1-14-1-security-update-advisory_{context}"]
+=== RHEA-XXXX:NNNN - {gitops-title} 1.14.1 security update advisory
+
+Issued: 2024-XX-YY
+
+The list of security fixes that are included in this release are documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHEA-XXXX:NNNN[RHEA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="fixed-issues-1-14-1_{context}"]
```

Comment:
Shall we add RN for https://issues.redhat.com/browse/GITOPS-4234 and https://issues.redhat.com/browse/GITOPS-5640

cc: @keithchong @jparsai

Reply:
For 4234, we discussed this in the standup. It will improve performance and there is no functional change, so lets not RN it.